docs/table_documentation.py

```diff
@@ -27,9 +27,10 @@ def __init__(self, component: str, tgt_fldr: str):
         self.stub = "data_model"
         # Get the appropriate data for the database we are documenting
         self.conn = database_connection(db_type=folder, project_path=self.proj_path)
-        self.path = join(*Path(realpath(__file__)).parts[:-1],
-                        f"../aequilibrae/project/database_specification/{folder}/tables")
-        self.doc_path = str(Path(realpath(__file__)).parent / "source" / tgt_fldr)
+        self.path = join(
```

Comment:
To those that make sense, we need to put links back to the technical documentation